src/components/shared/Tooltip.css

```diff
@@ -60,13 +60,13 @@
 }
 
 .tooltipLabel {
-  color: #666;
+  color: var(--grey-50);
   white-space: nowrap;
   text-align: right;
 }
 
 .tooltipBackTraceTitle {
-  color: #666;
+  color: var(--grey-50);
```

Comment:
Bring some consistency between colors in Tooltips and colors in the Sidebar.
Also, the grey that was used weren't constrasted enough for accessibility.